lib/scss/components/news-card/_card--horizontal.scss

```diff
@@ -1,52 +0,0 @@
-@charset "UTF-8";
-
-//
-// Horizontal Card
-//
-
-.su-card--horizontal {
```

Comment:
@boznik @jenbreese I couldn't find where this is being used within this module. Was this just leftover copy pasta from something?